include/site-health.php

```diff
@@ -29,7 +29,7 @@ function notice_site($url, $check_health=false)
     $entry = $result[0];
     
     //If we are allowed to do health checks...
-    if(!!$check_health){
+    if($check_health){
```

Comment:
Yes, I didn't personally touch this file, and neither did @Quix0r apparently. I'll do a sitewide pass right after master is up-to-date.